src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.module.css

```diff
@@ -0,0 +1,85 @@
+.mainContainer {
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+  gap: 2rem;
+  overflow-y: auto;
+  padding-top: 0.5rem;
+  padding-bottom: 3rem;
+}
+
+.subContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding-left: 0.25rem;
+  padding-right: 0.25rem;
+}
+
+.title {
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  font-size: 1.125rem;
+  font-weight: 600;
+}
+
+.errorGroupContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 0.25rem;
+  overflow-x: auto;
+  border-radius: 0.375rem;
+  background-color: var(--testsuites-v2-error-bg);
+  padding-left: 0.75rem;
+  padding-right: 0.75rem;
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
+}
+
+.errorToggleButton {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  gap: 0.25rem;
+}
+
+.errorCountLabel {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0.375rem;
+  background-color: var(--testsuites-failed-color);
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  padding-top: 0;
+  padding-bottom: 0;
+  font-size: 0.75rem;
+  font-weight: 700;
+  color: #ffffff;
+}
+
+.errorSummary {
+  margin-left: 0.5rem;
+  flex-grow: 1;
+  text-align: left;
+  overflow: hidden;
```

Comment:
This is missing `white-space: nowrap` and `text-overflow: hidden` to match up to the `truncate` tailwind class. Unless you wanted to keep multiline text for the summary and then we can drop the `overflow: hidden` but then have to fix the count and icon alignment.

<img width="431" alt="image" src="https://github.com/replayio/devtools/assets/788456/df486a29-e994-4bee-ae7d-2a17f268027d">

Reply:
Whoops, oversight. Thanks!